sources/packages/web/src/composables/useFormioUtils.ts

```diff
@@ -261,6 +280,32 @@ export function useFormioUtils() {
     customUtils[name] = method;
   };
 
+  /**
+   * Generate the dynamic form definition removing unwanted properties.
+   */
+  const getReadyToSaveFormDefinition = (formDefinition: unknown): unknown => {
+    // Clone the form definition to avoid modifying the original one.
+    const clonedForm = JSON.parse(JSON.stringify(formDefinition));
+    // Remove non-required properties from the form definition.
+    NON_REQUIRED_FORM_PROPERTIES.forEach((property) => {
+      delete clonedForm[property];
+    });
```

Comment:
What about the same elimination at the server side?

Reply:
These properties are not harmful, they do not pose a security risk (IMO), and we remove them because we do not use them.
I would avoid executing the same logic twice. It is important on the client side because the definition can be saved and persisted directly from the client side to the repo, for instance.
I did not see a need to replicate the same on the server side. Please let me know if we need to extend this conversation or if I am missing something.

Reply:
Okay. Given that the endpoint is ministry only I am good with not replicating the logic on server side.